README.md

```diff
@@ -1,8 +1,15 @@
 # SupernovaSDK-examples
-In the "scripts" folder:
 
-Examples of how to sequence operations with the SupernovaSDK using TransferController
+Welcome to the SupernovaSDK examples repository! This public repository provides a collection of examples intended to show the users how the API of the [Supernova SDK](https://pypi.org/project/BinhoSupernova/) works.
 
-In the "notebooks" folder:
+### Jupyter Notebooks
 
-Explore Jupyter notebooks for testing Supernova features and sensor setups.
+The [notebooks](./notebooks/) folder hosts different Jupyter notebooks sorted in sub folders by protocol or interface. [Jupyter Notebooks](https://jupyter.org/) are an interactive user interface that allows user to run code in cells. Maybe you do not know the name Jupyter Notebooks but you may have used [Google Colab](https://colab.google/) instead, which is based on Jupyter Notebooks. Finally, there is also a [Visual Studio Code extension for Jupyter Notebooks](https://code.visualstudio.com/docs/datascience/jupyter-notebooks).
```

Comment:
The [notebooks](./notebooks/) folder hosts different Jupyter notebooks sorted in sub folders by protocol or interface. [Jupyter Notebooks](https://jupyter.org/) are an interactive user interface that allows user to run code in cells. Maybe you do not know the name Jupyter Notebooks but you may have used [Google Colab](https://colab.google/) which is based on Jupyter Notebooks or used the [Visual Studio Code extension for Jupyter Notebooks](https://code.visualstudio.com/docs/datascience/jupyter-notebooks).